rust/arrow/src/datatypes/field.rs

```diff
@@ -39,6 +39,12 @@ pub struct Field {
     metadata: Option<BTreeMap<String, String>>,
 }
 
+impl Default for Field {
+    fn default() -> Self {
+        Field::new("", DataType::Null, false)
+    }
+}
+
```

Comment:
Why should a default field be the Null field? My feeling is that should not have a default for `Field`.

Reply:
Seemed sensible to me, but it's not important so I've removed this.